src/redux/books/bookReducer.js

```diff
@@ -1,13 +1,48 @@
-import * as Actions from './bookAction';
+import { v4 as uuidv4 } from 'uuid';
+import * as actions from '../../action/bookAction';
 
-const bookReducer = (state = {}, action = {}) => {
+export const initialBooks = [
+  {
+    id: uuidv4(),
+    title: 'Book One',
+    author: 'author',
+    genre: 'Fiction',
+  },
+  {
+    id: uuidv4(),
+    title: 'Book Two',
+    author: 'author',
+    genre: 'Economy',
+  },
+  {
+    id: uuidv4(),
+    title: 'Book Three',
+    author: 'author',
+    genre: 'Documentation',
+  },
+];
```

Comment:
- [ ] As per the project's requirement the initial state of the books object is an empty array. Kindly make changes here. :pray:

Reply:
- Alright Noted

Reply:
This issue is well addressed! :+1: